OpenSim/Simulation/Model/ExternalLoads.h

```diff
@@ -84,7 +84,7 @@ OpenSim_DECLARE_CONCRETE_OBJECT(ExternalLoads,
     /* If point of applications for external forces must be re-expressed
        then build new storages to be assigned to the individual ExternalForces
        with the transformed point data. Hang-on to them so we can delete them. */
-    ArrayPtrs<Storage> _storages;
+    std::vector<std::shared_ptr<Storage>> _storages;
```

Comment:
I think this should be `SimTK::ClonePtr` or `SimTK::ResetOnCopy`.

Reply:
I used `shared_ptr` because I didn't want to re-specify the data source after copying or pay the price of copying potentially large data sets.  It this case it is the `ExternalLoads` itself that is creating the new data source based on transforming the point of force application. This should be an operation done outside of `ExternalLoads` and the data source reset. It could then be easily reassigned to a clone, but as it stands now, the clone is added to the `Model` and it would require more widespread changes that I thought would be going even further beyond the scope of this PR. This could be addressed as part of a more general `ExternalLoads` refactor. As it stands, I don't think there is an issue of leaking.